Assets/Mirage/Runtime/NetworkManager.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace Mirage
 {
+    public enum NetworkManagerMode { Offline, ServerOnly, ClientOnly, Host }
```

Comment:
We could use a flag enum for this, since Host is just Server+Client

Reply:
I think it would be easier to work with as is

Reply:
or we can revisit in a different PR?